etcd/getting-started-with-etcd.md

```diff
@@ -30,6 +30,8 @@ etcd:
   initial_cluster_state:       new
 ```
 
+If you are unable to provision your machine using Container Linux configs, check out the [Setting up etcd v3 on Container Linux "by hand"][by-hand]
```

Comment:
If you are unable to provision your machine using Container Linux configs, refer to....